platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -475,6 +468,9 @@ int _odp_packet_tcp_chksum_insert(odp_packet_t pkt);
 int _odp_packet_udp_chksum_insert(odp_packet_t pkt);
 int _odp_packet_sctp_chksum_insert(odp_packet_t pkt);
 
+int packet_l4_chksum(odp_packet_hdr_t *pkt_hdr, odp_proto_chksums_t chksums,
```

Comment:
Function name should be "_odp_" prefixed when it's visible in symbol table (moved from static to non-static)